_opcodes/op109-bgee.html

```diff
@@ -1,6 +1,6 @@
 ---
 n: 109
-name: "State: Hold"
+name: "State: Paralyze"
```

Comment:
Why the name change? Does at least NI also call it this way?

Reply:
Yes, NI calls it _Paralyze_ (guess to differentiate it from op175)